pipeline/UnitTest.groovy

```diff
@@ -0,0 +1,34 @@
+def runTest(String library_dir){
+    def shareMethod = load("$library_dir/jobs/ShareMethod.groovy")
+    String label_name="unittest"
+    lock(label:label_name,quantity:1){
+        node_name = shareMethod.occupyAvailableLockedResource(label_name, [])
+        node(node_name){
+            deleteDir()
+            checkout scm
+            withCredentials([
+                usernamePassword(credentialsId: 'ff7ab8d2-e678-41ef-a46b-dd0e780030e1',
+                                 passwordVariable: 'SUDO_PASSWORD',
+                                 usernameVariable: 'SUDO_USER')
+                ]){
+                try{
+                    sh """#!/bin/bash -ex
+                    pushd $WORKSPACE
+                    echo $SUDO_PASSWORD | sudo -S ./HWIMO-TEST
```

Comment:
Please add comment to tell run HWIMO-TEST with sudo is a special case.
otherwise, the other PR Gate may feel confused when following this template.